lib/omniauth/strategies/auth0.rb

```diff
@@ -6,6 +6,7 @@
 require 'omniauth-oauth2'
 require 'omniauth/auth0/jwt_validator'
 require 'omniauth/auth0/telemetry'
+require 'omniauth/auth0/errors'
```

Comment:
This was already transiently included thanks to including `jwt_validator` two lines above this, which is why I didn't add this line in my PR. But it doesn't hurt either.